spec/spec_models.cr

```diff
@@ -83,6 +83,35 @@ require "uuid"
     table_name users
   end
 
+  class Character < Granite::Base
+    adapter {{ adapter_literal }}
+    table_name characters
+
+    primary character_id : Int32
+    field! name : String
+  end
+
+  class Courier < Granite::Base
+    adapter {{ adapter_literal }}
+    table_name couriers
+
+    primary courier_id : Int32, auto: false
+    field! issuer_id : Int32
+
+    belongs_to service : CourierService, primary_key: "owner_id"
```

Comment:
@Blacksmoke16 just to confirm, can you use a Symbol instead of a String for the value?
```crystal
belongs_to service : CourierService, primary_key: :owner_id
```

Reply:
Looks like `mysql` is failing.  :-(

Reply:
Yea, ill take a look.

Should be able to, just sets it and calls `.id` on them.

Reply:
@drujensen I figured it out, see 2e80d10.

I'm wondering why this didn't come up with like `Book` or any other model that used a `Int32` or `Int64` PK with `auto: false`.

I guess no other spec had it setup like I did this one time. :man_shrugging: